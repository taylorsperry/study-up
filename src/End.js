import React from "react"

const End = props => {
    return (
       <section className="quiz-card">
        <h2 className="correct">Great job!</h2>
        <p className="explanation">You've finished practicing JavaScript fundamentals ... for now.</p>
        <button className="keep-practicing" onClick={props.newPractice}>Review again</button>
       </section> 
    )
}
export default End