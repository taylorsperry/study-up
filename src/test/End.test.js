import React from "react";
import End from "../End";
import { shallow } from "enzyme"

describe("End", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<End />)
  })

  it("should match snapshot when all data is passed correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
})