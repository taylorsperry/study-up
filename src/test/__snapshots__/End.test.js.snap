// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`End should match snapshot when all data is passed correctly 1`] = `
<section
  className="quiz-card"
>
  <h2
    className="correct"
  >
    Great job!
  </h2>
  <p
    className="explanation"
  >
    You've finished practicing JavaScript fundamentals ... for now.
  </p>
  <button
    className="keep-practicing"
  >
    Review again
  </button>
</section>
`;
